workshops/notes_to_self/README.md

```diff
@@ -0,0 +1,677 @@
+# Notes to Self
+
+Short Link: [https://workshops.hackclub.com/notesToSelf/](https://workshops.hackclub.com/notesToSelf/)
```

Comment:
Needs to be updated to `notes_to_self`. The shortlink is based on the directory's name.